test/conformance/kuttl/rangeoperators/standard/02-resource.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+apply:
+- resource.yaml
```

Comment:
I would be surprised if this passes, apply the resource should hence the test step should fail too.
You probably want something like this:
```yaml
apply:
- file: resource.yaml
  shouldFail: true
```

Reply:
@eddycharly  this gives  "error converting TestStep:/ from unstructured error: unrecognized type: string"

Reply:
what version of kuttl are you using ?

Reply:
@eddycharly version 0.15.0

Reply:
@eddycharly maybe you should push on my PR?

Reply:
Ok, we forked kuttl, we needed features that did not exist upstream.

Reply:
Let me 👀

Reply:
@yimikao I updated the PR

Reply:
Thanks!

Reply:
@eddycharly how do i get the kuttl version that you use?

Reply:
It’s in makefile.

Reply:
You can `go install http://github.com/kyverno/kuttl/cmd/kubectl-kuttl@v0.0.0-20230108220859-ef8d83c89156`